zeek-cut/CMakeLists.txt

```diff
@@ -5,7 +5,3 @@ set(zeekcut_SRCS
 add_executable(zeek-cut ${zeekcut_SRCS})
 
 install(FILES zeek-cut.1 DESTINATION ${ZEEK_MAN_INSTALL_PATH}/man1)
-
-# Install wrapper script for Bro-to-Zeek renaming.
-include(InstallSymlink)
-InstallSymlink("${CMAKE_INSTALL_PREFIX}/bin/zeek-wrapper" "${CMAKE_INSTALL_PREFIX}/bin/bro-cut")
```

Comment:
Looks like this was taken care of by https://github.com/zeek/zeek-aux/commit/67f10f8e010f59b0913f289d1160657c70ccf0ed, if you want to rebase.